### PR TITLE
(DOCSP-11402): Remove repetition in Don't Humanize Inanimate Objects

### PR DESCRIPTION
**Pull request by melissamahoney-mongodb**
[JIRA](https://jira.mongodb.org/browse/DOCSP-11402)
[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-11402/style-guide/writing/use-effective-verbs.html#don-t-humanize-inanimate-objects)

### CODE REVIEW

File: source/style-guide/writing/use-effective-verbs.txt [hunk 1]
```diff
@@ -329,9 +329,7 @@ Be careful not to ascribe human feelings, motivations, and actions to
 inanimate objects. For example, a web application doesn't *know*,
 *need*, *remember*, *see*, *think*, *understand*, or *want*.
 
-Be careful not to ascribe human qualities, traits, or emotions to
-inanimate objects like software, hardware, or interfaces. Often, we use
-anthropomorphic words in an attempt to make difficult topics more understandable or relatable. This can actually make the
+Often, we use anthropomorphic words in an attempt to make difficult topics more understandable or relatable. This can actually make the
```
Comment (atsansone) [inline source/style-guide/writing/use-effective-verbs.txt:332]: **suggestion:** As written, this is hard to follow. I would recommend condensing.

```suggestion
Often, we use words that anthropomorphize (or humanize) to simplify topics. This can actually make the
```

Comment (melissamahoney-mongodb) [reply source/style-guide/writing/use-effective-verbs.txt:332]: Reworded.

File: source/style-guide/writing/use-effective-verbs.txt [hunk 2]
```diff
@@ -336,11 +336,11 @@ Be careful not to ascribe human feelings, motivations, and actions to
 inanimate objects. For example, a web application doesn't *know*,
 *need*, *remember*, *see*, *think*, *understand*, or *want*.
 
-Be careful not to ascribe human qualities, traits, or emotions to
-inanimate objects like software, hardware, or interfaces. Often, we use
-anthropomorphic words in an attempt to make difficult topics more understandable or relatable. This can actually make the
-concept harder to grasp for ESL readers or those less familiar with
-metaphors.
+Often, we use anthropomorphic words to make difficult topics more
```
Comment (atsansone) [inline source/style-guide/writing/use-effective-verbs.txt:339]: **suggestion:** The use of *anthropomorphic* was why I added "humanize" first so people could understand that the word means the same. Consider adding that back in.

Also, do they have to be relatable or understandable? Could they be one or the other?

File: source/style-guide/writing/use-effective-verbs.txt [hunk 3]
```diff
@@ -336,11 +336,11 @@ Be careful not to ascribe human feelings, motivations, and actions to
...
+understandable or relatable. These words can actually make the
+concept harder to grasp for:
```
Comment (atsansone) [inline source/style-guide/writing/use-effective-verbs.txt:341]:
```suggestion
Often, we humanize technology with anthropomorphic words.
Some writers believe this makes difficult topics more
understandable or relatable. These words can make the
concept harder to grasp for:
```
